Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang:1.16 AS builder
+WORKDIR /go/src/github.com/forbole/big-dipper
+COPY . ./
+RUN CGO_ENABLED=0 go build -o /build/bdjuno ./cmd/bdjuno
```

Instead of running this command, we should use `make build` instead and let the Makefile to take care of properly building the command. This will ensure that the binaries are always built using the same method. 

```suggestion
RUN make build
```

---

Do you have any recommendations for the base image in stage 2? Without CGO_ENABLED=0, I get the error: "standard_init_linux.go:228: exec user process caused: no such file or directory" in alpine.

---

I think this is due to the fact that you are using `golang:1.16` to build the binary, and then trying to run such binary on an alpine image. I would suggest using `golang:1.16-alpine` instead so that it links to the proper OS libraries